modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/ManagementToolbar/ManagementToolbarRight.tsx

```diff
@@ -64,10 +66,10 @@ const ManagementToolbarRight: React.FC<ManagementToolbarRightProps> = ({
 	buttons,
 	display = {columns: true},
 	columns,
-	disabled,
 	filterSchema,
 }) => {
 	const [{pin}, dispatch] = useContext(ListViewContext);
+	const [visible, setVisible] = useState(false);
```

Comment:
Would be good to add context to this state, I know it's visible, but basically anything could be visible here.

Maybe: `columnsDropdownVisible` clear than this